pyproject.toml

```diff
@@ -63,6 +63,7 @@ select = [
     "E",
     "F",
     "SIM1",
+    "UP034",
```

Comment:
I think it would be better to select `UP0` and then add unfixed rules to `ignore`. Then we can gradually remove the ignored rules one by one rather than add them to `select` list. Just we do for `SIM1`.

Reply:
What if new `UP0*` rules appear with new ruff versions, and all of a sudden CI starts failing?

Reply:
Oh, okay!